app/api/reviews/index.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express');
+
+const router = express.Router();
+
+const ProjectReviews = require('../../database/models/projectReviewsModel');
+
+router.post('/', async (req, res) => {
+  try {
+    const reviews = [req.body.likely_to_recommend, req.body.supporter_communication, req.body.provided_value];
+    let correctBool = true;
+    for (let i = 0; i < reviews.length; i += 1) {
+      if (!(reviews[i].isInteger() && reviews[i] >= 1 && reviews[i] <= 5)) {
+        correctBool = false;
+        break;
+      }
+    }
```

Comment:
You can simplify this a bit by using some functional programming concepts: [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) and [`reduce`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce); map the `reviews` array to an array of booleans (true if the element is an integer between 1 and 5 inclusive) and reduce it using the `and` operator. Or maybe you could [`filter`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) the `reviews` array by if the elements are *not* integers between 1 and 5 inclusive and then check length. No loop needed and you can make `correctBool` a constant.

```javascript
const validMetrics = reviews
  .map((r) => r.isInteger() && r >= 1 && r <= 5)
  .reduce((a,r) => a && r);

const validMetrics = reviews.filter((r) => !r.isInteger() || r < 1 || r > 5).length === 0;
```